lib/stripe/stripe_client.rb

```diff
@@ -9,19 +9,35 @@ module Stripe
   class StripeClient
     # A set of all known thread contexts across all threads and a mutex to
     # synchronize global access to them.
-    @thread_contexts_with_connection_managers = []
+    @thread_contexts_with_connection_managers = Set.new
```

Comment:
Updated this to be a Set so prevent duplicate thread contexts from being added.

Reply:
Good idea.